cumulusci/robotframework/pageobjects/ObjectManagerPageObject.py

```diff
@@ -82,15 +109,30 @@ def create_currency_field(self, field_name):
             message="Fields And Relationships page did not load in 1 min",
         )
 
+    @capture_screenshot_on_error
+    def create_text_field(self, field_name):
+        """Creates a text field by taking in the field name"""
+        self.selenium.wait_until_page_contains_element(text_locator, timeout=60)
+        self.selenium.click_element(text_locator)
+        self.selenium.wait_until_page_contains_element(next_button, 60)
```

Comment:
I don't think this wait is necessary, is it? Since you're already waiting for the text field, isn't it safe to assume that if the text field is on the page, so is the button?

Reply:
removed the wait